test/functional/services/dashboard/add_panel.ts

```diff
@@ -95,7 +96,7 @@ export class DashboardAddPanelService extends FtrService {
           continue;
         }
         await button.click();
-        await this.common.closeToastIfExists();
+
```

Comment:
I think the new toast closing is missing here: `await toasts.closeToastIfExists();`.

Reply:
Wow, great catch!

Reply:
Oh, actually, I don't think so.  It's because it goes away on it's own in this case.
Let me put through the flaky runner 50x to be sure.

Reply:
I think we shouldn't change the test logic as part of this PR unless we're absolutely sure that this code line should not be there.

Reply:
Ok, it's running now:

https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/4978